vertical-pod-autoscaler/pkg/admission-controller/logic/server.go

```diff
@@ -92,7 +97,7 @@ func (s *AdmissionServer) admit(data []byte) (*v1.AdmissionResponse, metrics_adm
 			response.Allowed = false
 		}
 	} else {
-		patches, err = nil, fmt.Errorf("not supported resource type: %v", admittedGroupResource)
+		patches, err = nil, fmt.Errorf("not supported resource type: %v/%v", admittedGroupResource.Group, admittedGroupResource.Resource)
```

Comment:
I think we can avoid that change compare to upstream the %v should give us the full information (just a format change, that is not really important in that case I would say)